### ISSUE TITLE
BufferError: memoryview: underlying buffer is not C-contiguous

### ISSUE THREAD

**Issue by mitar** [automl/auto-sklearn#290]
Even with auto-sklearn 0.2.0 this error still happens. Reproduction is simple (ignore the badness of training on whole dataset):

```
import autosklearn.classification
import sklearn.datasets
X, y = sklearn.datasets.load_digits(return_X_y=True)
automl = autosklearn.classification.AutoSklearnClassifier()
automl.fit(X, y)
```

`pip freeze` output:

```
auto-sklearn==0.2.0
ConfigSpace==0.3.6
Cython==0.25.2
docutils==0.13.1
joblib==0.11
liac-arff==2.1.0
lockfile==0.12.2
nose==1.3.7
numpy==1.12.1
pandas==0.20.1
psutil==5.2.2
pynisher==0.4.2
pyparsing==2.2.0
pyrfr==0.4.0
python-dateutil==2.6.0
pytz==2017.2
PyYAML==3.12
scikit-learn==0.18.1
scipy==0.19.0
six==1.10.0
smac==0.5.0
typing==3.6.1
virtualenv==15.0.1
```

**Comment by mfeurer** [comment]
Thanks for reporting this. I won't be able to provide a fix for that within the next one or two weeks. Do you maybe want to have a look into that and contribute a fix to auto-sklearn?

**Comment by mitar** [comment]
I am not yet sure if I am familiar enough with the code. I will see.

**Comment by mfeurer** [comment]
I think a good starting point would be #191. However, I don't know where the actual error happens, so you might need to change the position of the array operation.

**Comment by Coolnesss** [comment]
Hey, I'm getting this as well, on auto-sklearn 0.2. Any ideas for a temporary fix?

**Comment by mfeurer** [comment]
You could make your data C-contiguous before passing it into Auto-sklearn as showed in #191.

**Comment by mfeurer** [comment]
Will be fixed in the next release via PR #365.
